Description: Relax dependencies on errno and bingden
 Debian carries errno 0.2.4 for now, and bindgen 0.x. The
 package builds and works just fine with those versions, so
 let's relax the dependencies a bit.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Updated: Sat, 12 Aug 2023 21:40:58 +0200 werdahias@riseup.net
---
Index: loopdev/Cargo.toml
===================================================================
--- loopdev.orig/Cargo.toml
+++ loopdev/Cargo.toml
@@ -20,7 +20,7 @@ keywords = ["loop", "losetup"]
 license = "MIT"
 repository = "https://github.com/mdaffin/loopdev"
 [dependencies.errno]
-version = "0.2.8"
+version = ">= 0.2.4, < 0.4"
 
 [dependencies.libc]
 version = "0.2.105"
@@ -37,7 +37,7 @@ version = "1.0.68"
 [dev-dependencies.tempfile]
 version = "3.2.0"
 [build-dependencies.bindgen]
-version = "0.59.1"
+version = ">=0.59.1, <1.0"
 features = ["runtime"]
 default-features = false
 
